src/util/verto/ev.c

```diff
@@ -4346,6 +4346,11 @@ infy_cb (EV_P_ ev_io *w, int revents)
     {
       struct inotify_event *ev = (struct inotify_event *)(buf + ofs);
       infy_wd (EV_A_ ev->wd, ev->wd, ev);
+      if (ev->len > NAME_MAX) {
```

Comment:
This is third-party code, and we don't use it.  After PR #623 we won't compile it.  So we should leave this hunk out.